### DISCUSSION TITLE
Got error when tried to run example

### DISCUSSION THREAD

**Discussion by mfrow** [grafana/xk6-browser#730] [Code]
I got an error when tried to run the example:
1. Clone the xk6-browser repo
2. Run: go install go.k6.io/xk6/cmd/xk6@latest
3. Run: xk6 build --output xk6-browser.exe --with github.com/grafana/xk6-browser
4. Run the example: .\xk6-browser run examples\fillform.js
I got the following:
![image](https://user-images.githubusercontent.com/19708825/214800546-cbabfdc1-5085-40c7-9afc-a59e442f3a6a.png)

the same error occurs for other examples

**Comment by ArielKo1248** [comment]
I get the same thing when running in Windows 11 with a build done today

**Comment by ankur22** [comment]
Hi @mfrow and @ArielKo1248,

This is very likely to do with that fact that we've refactored the example scripts to work with `await`/`async` keywords. The latest version of `xk6-browser` on `main` will work with the latest example scripts. Try this:

1. Clone the xk6-browser repo
2. Run: go install go.k6.io/xk6/cmd/xk6@latest
3. Run: xk6 build --output xk6-browser.exe --with github.com/grafana/xk6-browser@main
4. Run the example: .\xk6-browser run examples\fillform.js

Note `@main` at the end of the 3rd step.

Let me know if that helps,
Ankur

**Reply by mfrow** [reply]
Its work now, thanks!
